apps/examples/wakaama_client/wakaama_client.c

```diff
@@ -99,38 +110,16 @@ static void prv_wakaama_start(void)
 	}
 }
 
-static int app_dhcp_main(void)
-{
-	uint32_t timeleft = 15000;
-	struct dhcpc_state state;
-	void * dhcp_handle;
-	int ret;
-
-	dhcp_handle = dhcpc_open(NET_DEVNAME);
-	ret = dhcpc_request(dhcp_handle, &state);
-	if (ret != OK) {
-			dhcpc_close(dhcp_handle);
-			return -1;
-	}
-
-	netlib_set_ipv4addr(NET_DEVNAME, &state.ipaddr);
-	netlib_set_ipv4netmask(NET_DEVNAME, &state.netmask);
-	netlib_set_dripv4addr(NET_DEVNAME, &state.default_router);
-
-	printf("IP address : %s ----\n", inet_ntoa(state.ipaddr));
-	return 1;
-}
-
 
-static int change_resource(FAR void *arg)
+static void change_resource(FAR void *arg)
 {
 	char argbuffer[20];
 	memset(argbuffer, 0x00, 20);
 	strcpy(argbuffer, "/3/0/11 7");
 	dm_lwm2m_change_client_resource(argbuffer);
 }
 
-static conn_cb linkUpEvent()
+static void linkUpEvent(void)
```

Comment:
I have some comments regarding this function,
1. isConnected is not Tizen RT style. g_is_connected is right  style.
2. here we check ret only once , direct checking is better
 if (dm_lwm2m_start_client != DM_ERROR_NONE)
3. resource_val using here??